Podfile.lock

```diff
@@ -1,13 +1,10 @@
 PODS:
-  - GHUnitIOS (0.5.8)
-  - OCMock (2.0.1)
+  - GCNetworkReachability (1.3.2)
 
 DEPENDENCIES:
-  - GHUnitIOS (~> 0.5.6)
-  - OCMock (~> 2.0.1)
+  - GCNetworkReachability (~> 1.3)
 
 SPEC CHECKSUMS:
-  GHUnitIOS: 0fe5a01896673a7768ee76158fbae02ad8fde853
-  OCMock: 60bfc5e6bc0ed592a6b7fb805b6c4d7108d37eb6
+  GCNetworkReachability: 2a7dd53d362220326b3e38df68c91b3ef84418d1
 
-COCOAPODS: 0.33.1
+COCOAPODS: 0.39.0
```

Comment:
nit; In general we should bump the version of cocoapods separately. If other developers are using different versions it can generate artifacts that lead to merge conflicts. I'm happy with using the latest version, though.